### DISCUSSION TITLE
Showcase university on GitHub profile

### DISCUSSION THREAD

**Discussion by jackTabsCode** [community/community#39452] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I think it would be cool for those of us that are in school the ability to put a University/College on our profile. Currently there is only an option to showcase your Company, but for those of us that are students, it would be cool to showcase that as well. They could even implement it with their already-existing student verification so that you need to be verified under that school to put it on your proifle.

**Comment by zhaifanhua** [comment]
Will this involve privacy?
